src/Microsoft.AspNetCore.Blazor.Templates/content/BlazorHosted.CSharp/BlazorHosted.CSharp.Server/BlazorHosted.CSharp.Server.csproj

```diff
@@ -7,6 +7,7 @@
       https://api.nuget.org/v3/index.json;
       https://dotnet.myget.org/f/blazor-dev/api/v3/index.json;
     </RestoreSources>
+    <LangVersion>7.3</LangVersion>
```

Comment:
This isn't strictly needed in this project, but it seemed like a better choice to be consistent.